prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -209,10 +209,10 @@ ssize_t rxr_pkt_init_cts(struct rxr_ep *ep,
 		cts_hdr->flags |= RXR_CTS_READ_REQ;
 
 	/* CTS is sent by different communication protocols.
-	 * CTS is sent using tx_entry in the emulated longcts read 
-	 * protocol. The emulated longcts write and the longcts 
+	 * CTS is sent using tx_entry in the emulated longcts read
+	 * protocol. The emulated longcts write and the longcts
 	 * message protocols sends CTS using rx_entry.
-	 * This check ensures appropriate tx_id and rx_id are 
+	 * This check ensures appropriate tx_id and rx_id are
```

Comment:
Personally don't mind this (and I know it's probably your editor/IDE), but someone may complain about the whitespace trimming being included in the commit - makes the Git blame a little weird I suppose.

Reply:
I wrote a separate commit for formatting changes (which is what I have been told to do in the past for formatting changes).  I think I put all the formatting changes into the correct commit, but if I messed that up, please let me know where.

Reply:
My b, I didn't catch that - I only ever look at the entire diff